File: Function/1.rb
def login(id)
    members = ["Albert", "Bethany", "Charles"]
    for member in members
        if member == id
            return true
        end
    end
    return false
end

print("Enter the ID: ")
input_id = gets.chomp()
if login(input_id)
    puts("Welcome, " + input_id + "!")
else
    puts("Who are you?")
end

File: Function/1.py
def login(id):
    members = ["Albert", "Bethany", "Charles"]
    for member in members:
        if member == id:
            return True
    return False

input_id = input("Enter the ID: ")
if login(input_id):
    print("Welcome, " + input_id + "!")
else:
    print("Who are you?")